pyoos/parsers/ioos/get_observation.py

```diff
@@ -27,13 +29,13 @@ def __new__(cls, element):
 
         if version == "1.0":
             from pyoos.parsers.ioos.one.get_observation import GetObservation as GO10
-            return super(IoosGetObservation, cls).__new__(GO10, element=root)
+            return super(IoosGetObservation, cls).__new__(GO10)
```

Comment:
I think this is ok... from what little testing I just did.

```python
#!/usr/bin/env python
# -*- coding: utf-8 -*-
from __future__ import print_function


class IoosGetObservation(object):
    def __new__(cls, element):
        return super(IoosGetObservation, cls).__new__(GetObservation)
    def __init__(self, element):
        print("My element is", element)


class GetObservation(IoosGetObservation):
    def __init__(self, element):
        super(GetObservation, self).__init__(element=element)

        print("GetObservation Version 1.0")

if __name__ == '__main__':
    IoosGetObservation('testing')
```

```
My element is testing
GetObservation Version 1.0
```

It looks like element doesn't go away and it gets passed to the `__init__` even though `__new__` discarded it.